docs/sources/reference/commandline/cli.md

```diff
@@ -1681,7 +1681,8 @@ deleted.
 
 You can remove an image using its short or long ID, its tag, or its digest. If
 an image has one or more tag or digest reference, you must remove all of them
-before the image is removed.
+before the image is removed. If -f and image's short or long ID is specified,
+untag all the names and remove the image.
```

Comment:
I would suggest this sentence for clarity: "*If the -f flag is used and the image's short or long ID is specified, the command will untag all tags matching the ID and then remove the image itself.*"